airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -151,9 +153,7 @@ def __init__(
         self.fargate_pod_execution_role_arn = fargate_pod_execution_role_arn
         self.fargate_selectors = fargate_selectors or [{"namespace": DEFAULT_NAMESPACE_NAME}]
         self.create_fargate_profile_kwargs = create_fargate_profile_kwargs or {}
-        self.aws_conn_id = aws_conn_id
-        self.region = region
-        super().__init__(**kwargs)
+        super().__init__(aws_conn_id=aws_conn_id, region=region, **kwargs)
```

Comment:
Is this right?  You have 'region_name' in the templatable fields, then pass `region` here.

Reply:
Nevermind, I guess it's a safe hedge against the operator pulling them out.